### ISSUE TITLE
sysdata: CPU varables don't work in the `format` string

### ISSUE THREAD

**Issue by torsava** [ultrabug/py3status#1709]
Sysdata module works nice out of the box, shows both CPU and Memory usage. But when I went to configure it and set `format` to it's [default value from the docs](https://py3status.readthedocs.io/en/latest/modules.html#sysdata), CPU related placeholders don't work at all. Memory shows accuratelly, but it stops being colored appropriately.

Here's the setting I used:

````
sysdata {
        format = "[\?color=cpu_used_percent CPU: {cpu_used_percent}%], [\?color=mem_used_percent Mem: {mem_used}/{mem_total} {mem_unit} ({mem_used_percent}%)]"
}
````

What I'm getting (no colors):
````
...|, Mem: 6.54/15.32 GiB (42.65%)|...
````

**Comment by lasers** [comment]
What `py3status --version` are you on? This is a recent change.

**Comment by lasers** [comment]
Alternative: You can use old threshold names which is `cpu` and `mem`.

**Comment by torsava** [comment]
> What `py3status --version` are you on? This is a recent change (EDIT: that hadn't been released).

py3status version 3.12 (python 3.7.2)

**Comment by torsava** [comment]
> EDIT: You can download latest [sysdata](https://github.com/ultrabug/py3status/blob/master/py3status/modules/sysdata.py) module -- Click RAW button.

Doesn't work with my version. :/

**Comment by torsava** [comment]
> EDIT: Is there a typo in your `{cpu_used_percent}` placeholder?

Not that I can see. Note that I copied it directly from the docs.

**Comment by torsava** [comment]
Ha! After looking into the code of my version, it seems the problem is simply that color and CPU placeholders have changed names. When I copied the default `format` from my version, everything works!

Apologies for filing an issue!

I would add to the docs a mention that these parameters have been changed. If that sounds good to you, I'll make a PR.

**Comment by lasers** [comment]
>Apologies for filing an issue!

No worries.

>I would add to the docs a mention that these parameters have been changed. If that sounds good to you, I'll make a PR.

You're on `3.12`. We're on `3.16`. Users upgrading to `3.16` with existing `format` will still work because the placeholders would be renamed to new ones. I tested this just now with `3.12` format. Placeholders and color thresholds works okay herre.

However, users copying formats from latest `git` docs and applying it to their `3.12` versions, I'm not sure how we can avoid that. It looks like we changed to `cpu_used_percent` in `3.13` right after your `3.12`. Reading docstring in modules instead of `readthedocs` will avoid this problem especially if we're not on latest version... Cheers.
